Makefile

```diff
@@ -1,125 +0,0 @@
-####### Compiler, tools and options
-
-CC       = cc
-LEX      = flex
-YACC     = yacc
-ARCHFLAGS=
-DEFINES  = -DDONATOR -DPROCESS_CC
-CFLAGS   = -g -pipe -Wall -W -Wno-address -Wno-unused-parameter -Wno-unused-label -Wno-unused-result -O2 $(DEFINES) $(ARCHFLAGS)
-LEXFLAGS =
-YACCFLAGS= -d
-INCPATH  = -I. $(INCLUDES)
-LINK     = $(CC) -g
-LFLAGS   = $(ARCHFLAGS)
-LIBS     =
-SHLIBS = -lavutil -lavformat -lavcodec -largtable2
-DESTDIR = /usr/local
-AR       = ar cq
-RANLIB   = ranlib -s
-QMAKE    = qmake
-TAR      = tar -cf
-GZIP     = gzip -9f
-COPY     = cp -f
-COPY_FILE= cp -f
-COPY_DIR = cp -f -r
-INSTALL_FILE= $(COPY_FILE)
-INSTALL_DIR = $(COPY_DIR)
-DEL_FILE = rm -f
-SYMLINK  = ln -sf
-DEL_DIR  = rmdir
-MOVE     = mv -f
-CHK_DIR_EXISTS= test -d
-MKDIR    = mkdir -p
-
-####### Output directory
-
-OBJECTS_DIR = ./
-
-####### Files
-
-OBJECTS = comskip.o \
-		platform.o \
-		mpeg2dec.o \
-		video_out_dx.o \
-		ccextratorwin/608.o \
-		ccextratorwin/ccextractor.o \
-		ccextratorwin/encoding.o \
-		ccextratorwin/general_loop.o \
-		ccextratorwin/myth.o
-DIST	   = comskip.pro
-TARGET   = comskip
-
-####### Platform specific
-
-ifneq (,$(findstring Windows,$(OS)))
-	PLATFORMLIBS = -L./vendor -lgdi32 -lcomdlg32
-	PLATFORMINCS = -I./vendor
-	OBJECTS += win32_pthread.o
-else
-	UNAME_S := $(shell uname -s)
-	ifeq ($(UNAME_S),Linux)
-		SHLIBS += -pthread -lm
-	endif
-endif
-
-ifneq (,$(DEBUG))
-	CFLAGS += -ggdb -O0
```

Comment:
I was planning on it, but then came across this approach, which I think makes more sense: http://stackoverflow.com/a/4680578

We could add a "./build_debug" script that looks like this:

    ./configure --enable-donator CPPFLAGS=-DDEBUG CFLAGS="-ggdb3 -O0" && make

This will make things easy for users with gcc/gdb, while still allowing users of other compilers/debuggers to manually specify their preferred debugging flags.